packages/satusehat/src/Adaptor.js

```diff
@@ -34,17 +42,22 @@ export function execute(...operations) {
 /**
  * Make a GET request to Satusehat
  * @public
- * @example  <caption>Make a GET request to get an Organization</caption>
- * get("Organization", {"name": "somename"})
+ * @example  <caption>Make a GET request to get an Organization by Id</caption>
+ * get("Organization/:id")
  * @example <caption>Search for a Patient using their NIK number</caption>
  * get('/Patient', {
  *   identifier:'https://fhir.kemkes.go.id/id/nik|9271060312000001'
  * });
+ * @example <caption>Get a specific Encounter by subject</caption>
+ * get('/Encounter', {
+ * subject:100000030009  // This gets an encounter that maps to a specific patient whose id is:100000030009
+ * });
  * @function
  * @param {string} path - Path to resource
- * @param {object} params - Optional request params such as name.
+ * @param {object} params - Optional request params such as identifier.
```

Comment:
The description needs to be like `Object of query parameters to include in the request`. Because it's not clear that this parameter represents query parameters, and it's not clear that `{ identifier: 'joe' }` maps to `/Organization?identifier=joe`.

That mapping between the `params` object and the actual query parameters is one of the missing pieces that we're trying to address in this work generally.